integration/admin-client/src/main/java/org/keycloak/admin/client/resource/AuthenticationManagementResource.java

```diff
@@ -179,20 +179,43 @@ public interface AuthenticationManagementResource {
     @POST
     void lowerRequiredActionPriority(@PathParam("alias") String alias);
 
+    /**
+     * This endpoint available since Keycloak server 25
+     *
+     * @param alias
+     * @return
```

Comment:
Perhaps this should also contain a description for the parameters and the returned value? As well as for all other similar comments.

Reply:
True.

I was focusing mostly on the compatibility aspect and originally did not added javadoc for the endpoints, which did not already had it. But perhaps, it is better to add javadoc when on this :-) So added it.